test/fixtures/m3u8/llhlsDelta.json

```diff
@@ -172,7 +173,7 @@
     "CAN-BLOCK-RELOAD": true,
     "CAN-SKIP-UNTIL": 12,
     "PART-HOLD-BACK": 1,
-    "HOLD-BACK": 2
+    "HOLD-BACK": 12
```

Comment:
targetDuration is 4 for this source, which means HOLD-BACK has to be at least 12.